tests/Application/src/Entity/Route/ShowBookWithLegacyAttribute.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace App\Entity\Route;
+
+use App\Entity\Book;
+use JMS\Serializer\Annotation as Serializer;
+use Sylius\Component\Resource\Annotation\SyliusRoute;
+
+/**
+ * @Serializer\ExclusionPolicy("all")
+ */
+#[SyliusRoute(
+    name: 'show_book_with_legacy_attribute',
+    path: '/book/{id}',
+    controller: 'app.controller.book:showAction',
```

Comment:
Symfony uses `::` since SF5 if it's applicable here
```suggestion
    controller: 'app.controller.book::showAction',
```